app/MwApi/MwApiExtras.php

```diff
@@ -63,8 +63,13 @@ public static function sendEmail($wiki, $username, $title, $content)
      * @param  string $key - to allow additional types of blocks (only 3 really exist though: bkusers, bkip, bkids)
      * @return array $response - the block information that comes up
      */
-    public static function getBlockInfo($wiki, $target, $key = null)
+    public static function getBlockInfo($wiki, $target, $key = null, $appealid)
```

Comment:
Optional parameters should come after required ones, also please add `$appealid` to the documentation block